### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by wattli**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,148 +1,28 @@
...
+Group alias: istio-security@googlegroups.com
 
-## Overview
+Here are the general structure for Istio Security SIG:
 
-Istio Auth's aim is to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+(1) Istio Security SIG
+Leads: Wencheng Lu (Google), Etai Lev-Ran (IBM), Michael Elder (IBM)
+Scopes:
+General Istio security topics across all sub workgroups, like go over Istio 0.3 security tasks
```
Comment (myidpt) [inline README.md]: tasks.

File: README.md [hunk 2]
```diff
@@ -1,148 +1,28 @@
-# Istio Authentication
+# Istio Security
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/istio/auth)](https://goreportcard.com/report/github.com/istio/auth)
 [![codecov](https://codecov.io/gh/istio/auth/branch/master/graph/badge.svg)](https://codecov.io/gh/istio/auth)
 
+Group alias: istio-security@googlegroups.com
 
-## Overview
+Here are the general structure for Istio Security SIG:
```
Comment (myidpt) [inline README.md]: Istio Security SIG and working groups

File: README.md [hunk 3]
```diff
@@ -1,148 +1,28 @@
-# Istio Authentication
+# Istio Security
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/istio/auth)](https://goreportcard.com/report/github.com/istio/auth)
 [![codecov](https://codecov.io/gh/istio/auth/branch/master/graph/badge.svg)](https://codecov.io/gh/istio/auth)
```
Comment (myidpt) [inline README.md]: Do you think we need more general contents to redirect to other pages, such as architecture (authentication.md), contributing (contributing.md) and testing (I can add this page)?

**Comment by wattli** [comment]
Seems there is some discussions ongoing about the doc and I will hold off the submission of this PR.

File: README.md (hunk 3 above)
Comment (wattli) [reply README.md]: Sure, that would be great.

File: README.md (hunk 2 above)
Comment (wattli) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (wattli) [reply README.md]: good point, done.

File: README.md [hunk 4]
```diff
@@ -1,148 +1,28 @@
...
 
+Group alias: istio-security@googlegroups.com
 
-## Overview
+Istio Security SIG and sub workgroups:
 
-Istio Auth's aim is to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+(1) Istio Security SIG
```
Comment (myidpt) [inline README.md]: Using bullets instead of (1), (2)... can give you better indention.

Comment (wattli) [reply README.md]: i see, thanks, done.

**Pull request merged**